### PR TITLE
First draft of blog for Admirer

### PR DESCRIPTION
**Pull request by andrewhinh**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Briefly describe the changes you have introduced.

### CODE REVIEW

File: _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+author: "The Transformees: Andrew Hinh and Aleks Hiidenhovi(Guest post)"
```
Comment (strickvl) [inline _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md:3]:
```suggestion
author: "The Transformees: Andrew Hinh and Aleks Hiidenhovi (Guest post)"
```

File: _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The second is creating data, training, and testing pipelines with both Makefiles and ZenML pipelines. This allows us to more easily iterate through ideas to improve the pipeline's performance and add features such as continual training, a feature of MLOps projects that ZenML goes into detail [here](https://blog.zenml.io/ci-ct-cd-with-zenml/). In addition, it allows others to more easily reproduce and test our work and us to both
```
Comment (strickvl) [inline _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md:52]:
```suggestion
The second is creating data, training, and testing pipelines with both Makefiles and ZenML pipelines. This allows us to more easily iterate through ideas to improve the pipeline's performance and add features such as continual training, a feature of MLOps projects that ZenML goes into detail [here](https://blog.zenml.io/ci-ct-cd-with-zenml/). In addition, it allows others to more easily reproduce and test our work.
```

File: _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+The second is creating data, training, and testing pipelines with both Makefiles and ZenML pipelines. This allows us to more easily iterate through ideas to improve the pipeline's performance and add features such as continual training, a feature of MLOps projects that ZenML goes into detail [here](https://blog.zenml.io/ci-ct-cd-with-zenml/). In addition, it allows others to more easily reproduce and test our work and us to both  
+
+
+## 💭 Conclusion
+Having recently built a MVP of the project for the Full Stack Deep Learning course, we were excited to build out the data management and training pipelines for this competition. We were not disappointed: we got a chance to get our hands dirty building out the pipelines, showcase our work to numerous experts in the field and receive their feedback, and win the "Most Promising Entry" award! We'd like to thank the ZenML team for organizing this competition and the judges for their invaluable feedback.
```
Comment (strickvl) [inline _posts/2022-12-19-month-of-mlops-most-promising-entry-prize-admirer.md:56]:
```suggestion
Having recently built a MVP of the project for the [Full Stack Deep Learning course](https://fullstackdeeplearning.com/), we were excited to build out the data management and training pipelines for this competition. We were not disappointed: we got a chance to get our hands dirty building out the pipelines, showcase our work to numerous experts in the field and receive their feedback, and win the "Most Promising Entry" award! We'd like to thank the ZenML team for organizing this competition and the judges for their invaluable feedback.
```

**Review by strickvl — CHANGES_REQUESTED** [review]
A couple of small changes that I can see, but otherwise looks good to me!

**Comment by htahir1** [comment]
@andrewhinh Thanks for the review! We'll try to ge tthis out tomorrow if you can accept the reviewsuggestions by @strickvl

**Comment by andrewhinh** [comment]
Accepted the review suggestions, thanks for the feedback!

**Review by strickvl — APPROVED** [review]

**Comment by htahir1** [comment]
@andrewhinh We are now live! We will be promoting now through the day. Please feel free to repost on social media!

**Pull request merged**